symmetric-db/src/main/java/org/jumpmind/db/sql/Row.java

```diff
@@ -115,7 +115,7 @@ public Date dateValue() {
     public Long longValue() {
         Object obj = this.values().iterator().next();
         if (obj != null) {
-            if (obj instanceof Long) {
+            if (obj instanceof Long || obj instanceof Double) {
```

Comment:
This looks like it will get an exception of "java.lang.ClassCastException: java.lang.Double cannot be cast to java.lang.Long".

Reply:
oh yeah, i think it would. let me fix that